src/draw/lv_draw_buf.h

```diff
@@ -43,11 +43,11 @@ typedef struct {
  * Refine it to suit your needs.
  */
 
-#define _LV_DRAW_BUF_STRIDE(w, cf) \
+#define LV_DRAW_BUF_STRIDE(w, cf) \
```

Comment:
This macro should be a function, macros are not accessible from high level language bindings

Reply:
It's a macro to allow allocating array with its return value in compile time. E.g. `static uint8_t buf[LV_DRAW_BUF_STRIDE(100, LV_COLOR_FORMAT_ARGB8888) * 20]`. I believe we cannot do such a thing in MP.